### ISSUE TITLE
[FE] Contact information: Remove requirement for international postal code for international addresses

### ISSUE THREAD

**Issue by Samara-Strauss** [department-of-veterans-affairs/va.gov-team#39775]
### Background

On 4.7.22, we learned about a veteran who was not able to save their address on VA.gov because they live in a country that does not have postal codes. The VA.gov profile currently requires international postal codes even though they are not required by the VA Profile backend. We should remove this requirement and make that field optional.

This is what the form looks like for international addresses. It is slightly different from the form we use for USA addresses. **We will still require zip codes for all USA addresses.**

![image](https://user-images.githubusercontent.com/34068740/162493742-753e5cae-7e64-4e83-84f5-8c1ef736f55d.png)

### Tasks

- [x] Remove requirement for international postal code from the international address form
- [x] Check in with Tom to see if there are any backend changes you both should sync up on. Backend investigation into potential next steps is happening in #39776.
- [x] Once the change is up on staging, have Samara validate the functionality -- **this needs to be validated before going to production**
- [x] Once validated, this can be pushed to prod

### Acceptance criteria

- [ ] We can successfully save an international address without an international postal code

**Comment by Samara-Strauss** [comment]
@mdewey when you get a chance, can you put an estimate on this ticket and then remove the "needs grooming" label? I've added this for you this sprint. Let me know if you have questions! I'm happy to regroup any time.

**Comment by mdewey** [comment]
Merged into staging, behind a flip. Waiting on BE ticket #39776 to continue. 

Even with the existing business rules of `address 1 AND (Country OR city OR zip)` There is a validation error coming from the address_validation endpoint. 

![Screen Shot 2022-04-20 at 10 04 48 AM](https://user-images.githubusercontent.com/1793923/164248838-2a97a811-fde4-499c-8c8e-f66ad4d6c09b.png)

![Screen Shot 2022-04-20 at 10 04 41 AM](https://user-images.githubusercontent.com/1793923/164248846-b226b7b3-0e8c-4122-bc7d-e041810205f9.png)


On hold until BE support is available

**Comment by Samara-Strauss** [comment]
Thanks for the update, @mdewey! We'll look to test the update once you can get some help from Tom to figure out the address validation issue.

**Comment by mdewey** [comment]
@Samara-Strauss This is now in staging! 

Tom and I have both checked it out and were able to save a foreign address without a zip code.

**Comment by Samara-Strauss** [comment]
Great! Validated with user 36. This should be ready to go to prod  🎉

**Comment by mdewey** [comment]
How does that process work? Do we want to toggle it on for a subset of users or just push it to everyone?

**Comment by mdewey** [comment]
Cool deal! 

When do want to go?

(sorry for the questions, learning the process :-) )

**Comment by Samara-Strauss** [comment]
No apologies necessary! This is good to go out any time.
